otlp-to-lgtm/module.river

```diff
@@ -0,0 +1,134 @@
+argument "otlp_http_endpoint" {
```

Comment:
It looks like `otlp_http_endpoint` and `otlp_grpc_endpoint` are unused. Can those be added below or removed if not needed?